lightning/src/ln/interactivetxs.rs

```diff
@@ -14,6 +14,7 @@ use core::ops::Deref;
 use bitcoin::blockdata::constants::WITNESS_SCALE_FACTOR;
 use bitcoin::consensus::Encodable;
 use bitcoin::policy::MAX_STANDARD_TX_WEIGHT;
+use bitcoin::ScriptBuf;
 use bitcoin::{
 	absolute::LockTime as AbsoluteLockTime, OutPoint, Sequence, Transaction, TxIn, TxOut,
 };
```

Comment:
Nit: there is already a `use` line for `bitcoin::`
```suggestion
use bitcoin::absolute::LockTime as AbsoluteLockTime;
use bitcoin::{OutPoint, ScriptBuf, Sequence, Transaction, TxIn, TxOut};
```